fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_VerticalRevisionPage.js

```diff
@@ -81,25 +81,34 @@ class CDUVerticalRevisionPage {
+            case 'X': { // at altitude on the coded vertical angle in Altitude2 field and at altitude specified in Altitude1 field
                 altitudeConstraint = this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel);
                 break;
             }
-            case '+': {
-                altitudeConstraint = "+" + this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel);
+            case '+': // "at or above altitude specified in Altitude1 field"
+            case 'H': // Glide Slope altitude (MSL) specified in Altitude2 field and at or above altitude specified in Altitude1 field
+            case 'J': // Glide Slope Intercept Altitude specified in Altitude2 field and at or above altitude specified in Altitude1 field
+            case 'V': { // at altitude on the coded vertical angle in the Altitude2 field and at or above altitude specified in Altitude1 field
+                altitudeConstraint = `+${this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel)}`;
                 break;
             }
-            case '-': {
-                altitudeConstraint = "-" + this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel);
+            case 'C': { // at or above altitude specified in Altitude2 field
+                altitudeConstraint = `+${this.formatFl(Math.round(waypoint.definition.altitude2), transAltLevel)}`;
                 break;
             }
-            case 'B': {
+            case '-': // at or below altitude specified in Altitude1 field
+            case 'Y': { // at altitude on the coded vertical angle in Altitude2 field and at or below altitude specified in the Altitude1 field
+                altitudeConstraint = `-${this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel)}`;
+                break;
+            }
+            case 'B': { // at or above to at or below altitudes in Altitude1 field and Altitude2 field
                 if (waypoint.definition.altitude1 < waypoint.definition.altitude2) {
-                    altitudeConstraint = "+" + this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel)
-                        + "/-" + this.formatFl(Math.round(waypoint.definition.altitude2), transAltLevel);
+                    altitudeConstraint = `+${this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel)}/-${this.formatFl(Math.round(waypoint.definition.altitude2), transAltLevel)}`;
                 } else {
-                    altitudeConstraint = "+" + this.formatFl(Math.round(waypoint.definition.altitude2), transAltLevel)
-                        + "/-" + this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel);
+                    altitudeConstraint = `+${this.formatFl(Math.round(waypoint.definition.altitude2), transAltLevel)}/-${this.formatFl(Math.round(waypoint.definition.altitude1), transAltLevel)}`;
```

Comment:
This should be displayed as "WINDOW" until a VNAV profile altitude is available.